src/systemdhook.c

```diff
@@ -642,42 +661,48 @@ static int prestart(const char *rootfs,
 		}
 	}
 
-	if (bind_mount(id, CGROUP_SYSTEMD, systemd_path, true)) {
+	if (bind_mount(id, CGROUP_SYSTEMD, systemd_path, false)) {
 		pr_perror("%s: Failed to bind mount %s on %s", id, CGROUP_SYSTEMD, systemd_path);
 		return -1;
 	}
 
 	/*
-	   Mount the writable systemd hierarchy into the container
+	   Mount the writable systemd hierarchies into the container
```

Comment:
The following change looks much bigger than it really is.  I basically added a for loop going through the new array of constant strings and sending each one of those to get_process_cgroup_subsystem_path() rather than calling it just once for "name=systemd"